content/apply_org.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Join the Movement"
+heading = "Join the Organization for Ethical Source"
++++
+
+If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indication the membership level that you are interested in:
+
+* _Active Members_ are individuals who volunteer their time, expertise, and talent towards the work of the organization. Active members have voting rights. Membership fees for active members are waived.
+* _Supporting Members_ are individuals or organizations who support the work of the organization through their membership fees. Supporting members do not have voting rights. Annual membership fees are $50 in the US, £50 in the UK, €50 in the European Union, or the equivalent in their local currencies.
+* _Associate Members_ are noncommercial organizations whose activities are related to our mission. Associate members do not pay membership fees and do not have voting rights. 
+
+### Membership Application
+
+<div class="form-container form-container-card">
+  <form><!-- class="send_message_form message form" name="apply" method="post" action="/" data-modal-id="modal-notify" data-netlify="true" netlify-honeypot="bot-field"-->
+    <div class="form-group">
+      <label class="hidden">
+        <span>For bots only</span>
+        <input name="bot-field">
+        <input name="subject" value="OES Membership Application">
+      </label>
+    </div>
+    <div class="form-group">
+      <label>
+        <p>E-mail</p>
+        <input name="email" type="email" required>
+      </label>
+    </div>
+    <div class="form-group">
+    <label>
+      <p>Membership Type</p>
+      <input name="membership_level" type="radio" value="Active Member">Active Member<br />
+      <input name="membership_level" type="radio" value="Supporting Member">Supporting Member<br />
+      <input name="membership_level" type="radio" value="Associate Member">Associate Member<br />
+    </label>
+    </div>
+    <div class="form-group">
+      <label>
+        <p>Message, brief bio, and why you're interested</p>
+        <textarea name="message" required></textarea>
+      </label>
+    <fieldset>
+      <button type="submit" class="button">Send</button>
```

Comment:
We should add a checkbox here for compliance with GDPR.

- [ ] I agree to received updates and information about the ESO and our activities. We won't share your email address without your express permission ever.